routers/web/repo/pull.go

```diff
@@ -752,6 +752,21 @@ func UpdatePullRequest(ctx *context.Context) {
 			ctx.Flash.Error(flashError)
 			ctx.Redirect(ctx.Repo.RepoLink + "/pulls/" + fmt.Sprint(issue.Index))
 			return
+		} else if models.IsErrRebaseConflicts(err) {
```

Comment:
I think, "else" can be omitted
```suggestion
		}
		if models.IsErrRebaseConflicts(err) {
```